flow/designs/asap7/fifo_oneclock/constraint.sdc

```diff
@@ -0,0 +1,26 @@
+# https://gist.github.com/brabect1/7695ead3d79be47576890bbcd61fe426
+#
```

Comment:
@tspyrou This is the .sdc file that I would like to have "upgraded" so that it creates two clock trees for the mock CPU core: one clock tree on the "external" side of the async FIFO and one on the CPU core side of the FIFO.

Reply:
@oharboe I want to make sure I understand. Do you want to create two additional clocks, one which is outside of the fifo for the input signals and one which is outside of the fifo for the output signals?